_security/access-control/api.md

```diff
@@ -169,12 +179,17 @@ Introduced 1.0
 
 Retrieves one action group.
 
+```json
+GET _plugins/_security/api/actiongroups/<action-group>
+```
+{% include copy-curl.html %}
 
-#### Request
+#### Example request
```

Comment:
not sure if this was intended. I see the format to be `Request` and `Example Response` in other occurences. @cwillum Thoughts on reverting it to `Request`?

Reply:
Good catch, DC. Technically, this is an example request: it contains a real name for an action group. And the Example response contains the same action group name. All of these should have a proper example for the request and the response. In that case, they would be example requests and example responses. But the format here uses a generic filler <action-group> for most of these requests, which isn't a proper example. For consistency I'll change this back to Request. Thanks.